map.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
All the html/css/js files will be served through the flask app. So, this will (eventually) need to be moved there.